iib/workers/tasks/build.py

```diff
@@ -118,6 +118,18 @@ def _create_and_push_manifest_list(request_id, arches):
     """
     buildah_manifest_cmd = ['buildah', 'manifest']
     output_pull_spec = get_rebuilt_image_pull_spec(request_id)
+
+    try:
+        run_cmd(
+            buildah_manifest_cmd + ['rm', output_pull_spec],
```

Comment:
I think this needs to be `rmi`, not `rm`.

Reply:
Nope, the docs say it has to be `rm`. https://github.com/containers/buildah/blob/main/docs/buildah-manifest-rm.md

Reply:
Unless, I am missing something

Reply:
Thanks for the correction :)